native/cocos/math/Quaternion.h

```diff
@@ -432,6 +441,24 @@ class CC_DLL Quaternion {
     static void slerpForSquad(const Quaternion &q1, const Quaternion &q2, float t, Quaternion *dst);
 };
 
+/**
+ * Quaterion linear interpolation.
+ */
+inline Quaternion lerp(const Quaternion& from, const Quaternion& to, Quaternion::ValueType t) {
+    Quaternion result;
+    Quaternion::lerp(from, to, t, &result);
+    return result;
+}
```

Comment:
It is not a good idea to implement it in cocos name space. It is easy to cause conflict. It is better to implement as a static function.

Reply:
More better is to implement as a member function.